### DISCUSSION TITLE
wiperf probe connect to influxdb via https

### DISCUSSION THREAD

**Discussion by robdev6** [wifinigel/wiperf#68] [Bugs - I think I may have found a bug] [answered]
Hi, 
i want to tranfer the data of the wiperf probe to an existing influxdb instance.
the influxdb has https enabled only. 
It doen't matter, if i use the setting "influx_ssl" with True or False at the config.ini

/etc/wiperf/config.ini
       influx_ssl: True / False

i read all the time the errors in the wiperf_agent.log

xxxx-xx-xx xx:xx:xx,xxx - Probe_Log - INFO - Sending results data to Influx host: yyy.yyy.yyy.yyy, port: 8086, database: wiperf)
xxxx-xx-xx xx:xx:xx,xxx - Probe_Log - ERROR - Issue sending data to Influx: 400: Client sent an HTTP request to an HTTPS server.

is it possible to send the data to influxdb via https?
maybe ignore ssl check is the problem?

Connection check is fine:
xxxx-xx-xx xx:xx:xx,xxx - Probe_Log - INFO -   Checked interface route to : 192.168.10.15. Result: yyy.yyy.yyy.yyy dev wlan0 src yyy.yyy.yyy.yyy uid 0
xxxx-xx-xx xx:xx:xx,xxx - Probe_Log - INFO -   Mgt interface route looks good.
xxxx-xx-xx xx:xx:xx,xxx - Probe_Log - INFO -   Checking port connection to InfluxDB server yyy.yyy.yyy.yyy, port: 8086
xxxx-xx-xx xx:xx:xx,xxx - Probe_Log - INFO -   Port connection to server yyy.yyy.yyy.yyy, port: 8086 checked OK.

thanks in advance for your reply.
cheers
Rob

**Comment by wifinigel** [comment]
Hi Rob,

Yes I had a look at the code and it looks as though I have missed setting the 'ssl' flag in the code when accessing InfluxDB with the Influx client in my code.

Give me a few days and I'll try to get you a fix out. Apologies for the oversight and thanks for highlighting this.

All the best.

Nigel.

**Reply by robdev6** [reply]
Hi, 
Thank you very much! Awesome fast response, Great!

Cu
Robert

**Comment by wifinigel** [comment, marked as answer]
Hi Rob, I've just added a new release that hopefully fixes your issue.

One thing to note is that the influx_ssl field needs to be "true", not "True". The parameter is case-sensitive.

I have tested it on an instance of Influx running https and it seems to work OK.

Check out this page for the upgrade process:

[(https://wifinigel.github.io/wiperf/probe_upgrade/#upgrade-to-latest-stable-release)]

Please let me know how it goes.

Nigel.

**Reply by wifinigel** [reply]
cc: @robdev6  (just to make sure you see this update :) )

**Reply by robdev6** [reply]
Hi, yes, i updated it and works perfect now!

thanks
Robert
